python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -377,6 +378,142 @@ class RandomForestClassifier(Base):
 
         return model_protobuf_bytes
 
+    def _tl_model_handles(self, model_bytes):
+        cdef ModelHandle cuml_model_ptr = NULL
+        mod_had_val = tl_mod_handle(& cuml_model_ptr,
+                                    <vector[unsigned char] &> model_bytes)
+        mod_handle = <size_t> cuml_model_ptr
+        print(" mod_handles in pyx _tl_model_handles : ", ctypes.c_void_p(mod_handle).value)
+
+        return ctypes.c_void_p(mod_handle).value #, <size_t> mod_had_val
+
+
+    def _read_mod_handles(self, mod_handles):
+
+        print(" mod_handles in pyx : ", mod_handles)
+        cdef uintptr_t model_ptr = <uintptr_t> mod_handles
+        model_protobuf_bytes = save_model(<ModelHandle> model_ptr)
+
+        return model_protobuf_bytes
+
+    def get_algo(self, algo_str):
```

Comment:
These methods (`get_algo` and `get_storage_type`) exist in the FIL Python wrapper already.

Reply:
I was testing the ModelHandles and trying to see if it would be better to call the C++ fil function from the rf file or use the fil.pyx file. This will be removed in the final version, as this is at the moment excess code.